docs/section19/19-12.md

```diff
@@ -0,0 +1,36 @@
+# Section 19 - Rule 19-12         
+**Schema Version:** 0.0.23    
+**Mandatory Rule:** True    
+**Rule ID:** 19-12         
+**Rule Description:** The baseline system economizer high-limit shutoff shall be a dry-bulb fixed switch with set-point temperatures in accordance with the values in Table G3.1.2.7.  
+
+**Rule Assertion:** Options are Pass/Fail/NOT_APPLICABLE   
+**Appendix G Section:** G3.1.2.7       
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None    
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:**  
+1. HVAC systems modeled with air-side economizers in the B_RMI.   
+   
+
+**Function Calls:**  None
+## Rule Logic:   
+**Applicability Check 1 -  Check if Economizer was Modeled in the B_RMI**  
+- Get climate zone: `climate_zone = ASHRAE229.weather.climate_zone`  
+- For each hvac system in the B_RMI: `for hvac in B_RMI...HeatingVentilatingAirConditioningSystem:` 
+    - Check if the system was modeled with an economizer, if not then outcome is NOT_APPLICABLE, if it is then carry on: `if hvac.fan_system.AirEconomizer != Null:`    
```

Comment:
I think this RDS also needs to check that AirEconomizer.Type="temperature"

Reply:
@mkarpman You ask me to remove this in the comment above.

Reply:
Add temperature back here. Remove the temperature check from 19-10.

Reply:
I added temperature back here and removed it from 19-10. I changed 19-10 to does not equal FIXED_FRACTION (instead of equals TEMPERATURE) as the type since (in eQuest at least) FIXED_FRACTION = no economizer.